### PR TITLE
Improve documentation for authenticated_origin_pulls & authenticated_origin_pulls_certificate

### PR DESCRIPTION
**Pull request by Nmishin**
Fix for: https://github.com/cloudflare/terraform-provider-cloudflare/issues/2733

### CODE REVIEW

File: examples/resources/cloudflare_authenticated_origin_pulls/import.sh [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Authenticated Origin Pull configuration
-$ terraform import cloudflare_authenticated_origin_pulls_certificate.my_aop <zone_id>//
+$ terraform import cloudflare_authenticated_origin_pulls.my_aop <zone_id>//
```
Comment (jacobbednarz) [inline examples/resources/cloudflare_authenticated_origin_pulls/import.sh:2]:
```suggestion
$ terraform import cloudflare_authenticated_origin_pulls.example <zone_id>
```

File: examples/resources/cloudflare_authenticated_origin_pulls/import.sh [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 # Per-Zone Authenticated Origin Pull configuration
-$ terraform import cloudflare_authenticated_origin_pulls_certificate.my_per_zone_aop <zone_id>/<certificate_id>/
+$ terraform import cloudflare_authenticated_origin_pulls.my_per_zone_aop <zone_id>/<certificate_id>/
```
Comment (jacobbednarz) [inline examples/resources/cloudflare_authenticated_origin_pulls/import.sh:5]:
```suggestion
$ terraform import cloudflare_authenticated_origin_pulls.example <zone_id>/<certificate_id>
```

File: examples/resources/cloudflare_authenticated_origin_pulls/import.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
 
 # Per-Hostname Authenticated Origin Pull configuration
-$ terraform import cloudflare_authenticated_origin_pulls_certificate.my_per_hostname_aop <zone_id>/<certificate_id>/<hostname>
+$ terraform import cloudflare_authenticated_origin_pulls.my_per_hostname_aop <zone_id>/<certificate_id>/<hostname>
```
Comment (jacobbednarz) [inline examples/resources/cloudflare_authenticated_origin_pulls/import.sh:8]:
```suggestion
$ terraform import cloudflare_authenticated_origin_pulls.example <zone_id>/<certificate_id>/<hostname>
```

**Comment by jacobbednarz** [comment]
LGTM. can you please also run `make docs` locally to generate and commit the documentation artifacts too?

**Comment by Nmishin** [comment]
@jacobbednarz in the Authenticated Origin Pull configuration we need to have two // in the end, for make import work:
```
origin_pulls$ terraform import cloudflare_authenticated_origin_pulls.my_aop df066cb84f969ab8f4bc51b7f2eb2aa6
cloudflare_authenticated_origin_pulls.my_aop: Importing from ID "df066cb84f969ab8f4bc51b7f2eb2aa6"...
╷
│ Error: invalid id ("df066cb84f969ab8f4bc51b7f2eb2aa6") specified, should be in format "zoneID/certID/hostname"
│ 
│ 
╵

origin_pulls$ 
origin_pulls$ terraform import cloudflare_authenticated_origin_pulls.my_aop df066cb84f969ab8f4bc51b7f2eb2aa6//
cloudflare_authenticated_origin_pulls.my_aop: Importing from ID "df066cb84f969ab8f4bc51b7f2eb2aa6//"...
cloudflare_authenticated_origin_pulls.my_aop: Import prepared!
  Prepared cloudflare_authenticated_origin_pulls for import
cloudflare_authenticated_origin_pulls.my_aop: Refreshing state... [id=00b781a09f02ce30521eb094f58228ff]

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

**Comment by jacobbednarz** [comment]
that seems to be a bug in the `Import` then. if we need a way to denote the different types, we can look to update the import string to be something with the right identifiers.

**Comment by Nmishin** [comment]
Agree, I will check the`Import` when I have free time.

**Comment by jacobbednarz** [comment]
thanks @Nmishin 👏

**Pull request merged**